buffer.py

```diff
@@ -943,6 +969,8 @@ def wrapper(*args):
 
     @build_context_wrap
     def wheelEvent(self, event):
+        self.clear_synctex_indicator()
```

Comment:
After synctex_pos_y become global coordiante, we just need test in paintEvent, don't need write clear_synctex_indicator.